### PR TITLE
Adding base things for the homework

### PR DESCRIPTION
**Pull request by Martincho2003**

### CODE REVIEW

File: homework_04/15_Martin_Dinev/point.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        if abs(distance - (self.radius + circle.radius)) < 0.0001:
+            return RelativePosition.TOUCHING
+
+        #distance < self.radius + circle.radius:
+        if distance + circle.radius < self.radius or \
+            (distance + circle.radius) - self.radius < 0.0001:
+            return RelativePosition.SAME
+
+        return RelativePosition.INTERSECTING
+
```
Comment (Vic-Dim) [inline homework_04/15_Martin_Dinev/point.py:66]: It's nice you've added multiline comments, but the code is self-documenting even without them

File: homework_04/15_Martin_Dinev/test_point.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    """
+    Test if two circles are intersecting
+    """
+    circle1 = Circle(Point(0.0, 0.0), 5.0)
+    circle2 = Circle(Point(10.0, 0.0), 7.0)
+    assert circle1.find_relative_position(circle2) == RelativePosition.INTERSECTING
+
+    circle1 = Circle(Point(1.2, 3.1), 8.6)
+    circle2 = Circle(Point(13.5, 17.2), 16.1)
+    assert circle1.find_relative_position(circle2) == RelativePosition.INTERSECTING
```
Comment (Vic-Dim) [inline homework_04/15_Martin_Dinev/test_point.py:61]: I see you've spent some time tidying the tests by using pylint too. Good

**Review by Vic-Dim — APPROVED** [review]

File: .github/workflows/homework_04_15_martin_dinev.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        pip install -r requirements.txt
+
+    - name: Analysing the code with pylint
+      run: |
+        cd homework_04
+        cd 15_Martin_Dinev
+        pylint point.py
+        pylint test_point.py
+
+    - name: Run the tests
```
[the remark this answers is not in the document]
Comment (Martincho2003) [reply .github/workflows/homework_04_15_martin_dinev.yaml:39]: Maybe your idea is good and actually im gonna change it

**Review by Vic-Dim — APPROVED** [review]

File: homework_04/15_Martin_Dinev/point.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        distance = sqrt(power(circle.center.x_coordinate - self.center.x_coordinate, 2) + \
+            power(circle.center.y_coordinate - self.center.y_coordinate, 2))
+        if distance > self.radius + circle.radius:
+            return RelativePosition.NO_COMMON_POINTS
+
+        if abs(distance - (self.radius + circle.radius)) < 0.0001:
+            return RelativePosition.TOUCHING
+
+        #distance < self.radius + circle.radius:
```
Comment (GerganaRoeva) [inline homework_04/15_Martin_Dinev/point.py:60]: Този коментар нужен ли е така

**Review by GerganaRoeva — APPROVED** [review]
Обобщено, добра работа.

File: homework_04/15_Martin_Dinev/point.py (hunk 4 above)
Comment (Martincho2003) [reply homework_04/15_Martin_Dinev/point.py:60]: Е може и без него ама за по добро разбиране на кода, като го погледна след време

File: homework_04/15_Martin_Dinev/test_point.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
+"""
+File to test if everything is correct
```
[the remark this answers is not in the document]
Comment (Martincho2003) [reply homework_04/15_Martin_Dinev/test_point.py:2]: Мерси

**Pull request merged**